gradle-recommended-product-dependencies/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'groovy'
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'org.inferred.processors'
+apply from: "${rootDir}/gradle/publish-jar.gradle"
```

Comment:
If we're expecting to compile against this jar, should we also include revapi to catch regressions?